test/parallel/test-require-deps-deprecation.js

```diff
@@ -39,6 +39,15 @@ for (const m of deprecatedModules) {
   } catch (err) {}
 }
 
+// Instead of checking require, check that resolve isn't pointing toward
+// /node/deps, as user might already have node installed with acorn in
+// require.resolve range
```

Comment:
Two small nits: 

* `/node/deps` looks like a path, so maybe just `a build-in module`?
* Can you add `Ref: https://github.com/nodejs/node/issues/17148` to this comment or something like that? Give people a link so they can see the bug that this approach avoids that the simpler approach being replaced doesn't.

Reply:
True! It's done